Programowanie3/Assets/Scripts/Movement.cs

```diff
@@ -28,4 +37,11 @@ public void MoveWithVelocity(Vector3 moveDirection)
     {
         rb.velocity = moveDirection * moveSpeed;
     }
+
+    public void TempMovementSpeed()
+    {
+        moveSpeed = temp;
+        Debug.Log("zero");
+        Invoke("Return", resetTime);
+    }
```

Comment:
```suggestion
    public void SetTempMovementSpeed(float temp, float resetTime)
    {
        moveSpeed = temp;
        Debug.Log("zero");
        Invoke("Return", resetTime);
    }
```
Może tak? Z Set nazwa jest bardziej opisowa. Metody zazwyczaj powinny zawierać czasownik mówiący co robią.

Jak damy parametry do metody ( wywalimy pole  z góry), to będzie można jej użyć też do innych rzeczy. Np. do przyspieszania i zwalniania.
W przyszłości zrobimy lepszy system czasowych modyfikacji parametrów